portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -48,8 +51,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable(FetchOptions.Builder.withLimit(commentsRequested))) {
       String name = (String) entity.getProperty("name");
       String text = (String) entity.getProperty("text");
+      String mood = (String) entity.getProperty("mood");
 
-      String comment = name + " says...\n" + text;
+      String comment = name + " says...\n\n" + text + "\n\n Mood: " + mood;
```

Comment:
Please use `String.format` if longer than 3 segments.
https://stackoverflow.com/questions/925423/is-it-better-practice-to-use-string-format-over-string-concatenation-in-java

Reply:
Will do, thank you.